### PR TITLE
some additional utils for state-trie-migration

### PR DESCRIPTION
**Pull request by kianenigma**
while writing the doc for this, I came across these two as potential improvements.

The doc can be found here: https://hackmd.io/JagpUd8tTjuKf9HQtpvHIQ

### CODE REVIEW

File: frame/state-trie-migration/src/lib.rs [hunk 1]
```diff
@@ -701,6 +711,40 @@ pub mod pallet {
...
+			let _ = T::ControlOrigin::ensure_origin(origin)?;
+			SignedMigrationMaxLimits::<T>::put(limits);
+			Ok(())
+		}
+
+		/// Forcefully set the progress the running migration.
+		///
+		/// This is only useful in one case: the next key to migrate is too big to be migrated with
+		/// a signed account, in a parachain context, and we simply want to skip it. A reasonable
+		/// example of this would be `:code:`, which is both very expensive to migrate, and commonly
```
Comment (bkchr) [inline frame/state-trie-migration/src/lib.rs:730]: This would happen on the next runtime upgrade, but yeah it should be fine.

**Review by bkchr — APPROVED** [review]

File: frame/state-trie-migration/src/lib.rs [hunk 2]
```diff
@@ -470,6 +470,13 @@ pub mod pallet {
 	#[pallet::getter(fn auto_limits)]
 	pub type AutoLimits<T> = StorageValue<_, Option<MigrationLimits>, ValueQuery>;
 
+	/// The maximum limits that the signed migration could use.
+	///
+	/// If not set, not signed submission is allowed.
```
Comment (cheme) [inline frame/state-trie-migration/src/lib.rs:475]:
```suggestion
	/// If not set, no signed submission is allowed.
```

**Review by cheme — APPROVED** [review]
Looks good

**Comment by kianenigma** [comment]
bot merge

**Pull request merged**